clients/cobol-lsp-vscode-extension/src/services/snippetcompletion/cobolSnippets.json

```diff
@@ -519,42 +754,168 @@
+  },
+  "MULTIPLE FILE TAPE": {
+    "prefix": "MULTIPLE FILE TAPE",
+    "body": [
+      "FD ${1:QSAM-FILE}",
+      "\tLABEL RECORDS ARE STANDARD",
+      "\tMULTIPLE FILE TAPE.",
+      "\t${3:RPT-OUT-REC} PIC X(${4:132})."
+    ],
+    "description": "DATA DIVISION - The MULTIPLE FILE TAPE clause specifies that 2 or mote data files share the same physical VOL=SER of tape. https://publibfp.dhe.ibm.com/epubs/pdf/igy6lr40.pdf"
+  },
+  "FILE CONTROL": {
+    "prefix": "FILE CONTROL",
+    "body": [
+      "SELECT ${1:FILE-1} ASSIGN TO ${2:EXTFILE1}",
+      "ORGANIZATION IS ${3|INDEXED,RELATIVE,SEQUENTIAL,LINE-SEQUENTIAL|}",
+      "ACCESS MODE IS ${4|SEQUENTIAL,RANDOM,DYNAMIC|}",
+      "FILE STATUS IS ${5:WS-FILE-STATUS-VAR}."
+    ],
+    "description": "ENVIRONMENT DIVISION / INPUT-OUTPUT SECTION / FILE-CONTROL - The FILE-CONTROL paragraph associates every file in your COBOL program with an external data set. It optionally defines file organization, access mode, etc. https://publibfp.dhe.ibm.com/epubs/pdf/igy6lr40.pdf"
+  },
+  "FILLER": {
+    "prefix": "FILLER",
+    "body": [
+      "01 ${2:GROUP-ITEM}.",
+      "\t05 ${3:ELEMENTARY-LINE}",
+      "\t\t10 ${4:FIELD-1} PIC S9(${5:5}) COMP-3.",
+      "\t\t10 FILLER PIC X(${1:20}) VALUE SPACE.",
+      "\t\t$0"
+    ],
+    "description": "A placeholder variable that is not explicitly referred to in a program. FILLER is used to line up variable names with byte-offsets in the physical data record. https://publibfp.dhe.ibm.com/epubs/pdf/igy6lr40.pdf"
+  },
+  "FIRST #1": {
     "prefix": "FIRST",
     "body": ["FIRST ${1:id} BY ${2:id}"],
     "description": "FIRST ideral BY ideral"
   },
+  "FIRST #2": {
+    "prefix": "FIRST",
+    "body": [
+      "INSPECT ${4:VAR-1}",
+      "TALLYING ${5:VAR-CTR} FOR LEADING \"${6:A}\"",
+      "REPLACING FIRST \"${1:A1}\" BY \"${2:2}\" AFTER INITIAL \"${3:T}\""
+    ],
+    "description": "When FIRST is specified and not BEFORE / AFTER - the replacement field changes the left/first occurrence of the inspected field (VAR-1 in this example). https://publibfp.dhe.ibm.com/epubs/pdf/igy6lr40.pdf"
+  },
+  "FOOTING": {
+    "prefix": "FOOTING",
+    "body": ["WITH FOOTING AT ${1:literal}"],
+    "description": "Part of an FD's \"LINAGE\" (logical page size) clause - available for QSAM and LINE-SEQUENTIAL files only). WITH FOOTING specifies a line # in an output file or printer. https://publibfp.dhe.ibm.com/epubs/pdf/igy6lr40.pdf"
+  },
   "FOR": {
     "prefix": "FOR",
     "body": ["FOR ${1:item}"],
     "description": "FOR item"
   },
+  "FOR CHARACTERS AFTER": {
+    "prefix": "FOR CHARACTERS AFTER",
+    "body": [
+      "INSPECT ${2:variable-field} TALLYING ${3:num-field)",
+      "FOR CHARACTERS AFTER ${1:X\"40\"}"
+    ],
+    "description": "Specifies search characters - in INSPECT verb. https://publibfp.dhe.ibm.com/epubs/pdf/igy6lr40.pdf"
```

Comment:
This needs to be a bit clearer I think, how about
"Counts the number of characters after the first occurrence of a specified string. Part of the TALLYING keyword in the INSPECT statement."
(maybe it doesn't need to be a string, not sure)